API-Samples/Tutorial/markdown/10-init_render_pass.md

```diff
@@ -105,10 +111,28 @@ optimal layout so that the GPU can read it more efficiently.
 Conversions from one layout to another are called layout transitions
 in Vulkan.
 You have control over these transitions and can invoke them
-with an image memory barrier command or as a subpass dependency in a render
-pass.
-In this case, the sample code uses image memory barrier commands, which
-are detailed later in this section.
+in one of three ways:
+
+1. Memory Barrier Command (via `vkCmdPipelineBarrier`)
+1. Render Pass specification
```

Comment:
Could we say Render Pass final layout specification.  I know for a while I thought that specifying initialLayout in the renderpass definition would perform a layout transition for me (which it will not), so maybe spelling that out could avoid confusion